### ISSUE TITLE
[CoE Starter Kit - Who can help? - Solution Load Issue

### ISSUE THREAD

**Issue by billwhalenmsft** [microsoft/coe-starter-kit#2227]
### What is your question?

Hello,

I am a TSP for Microsoft and am trying to load the core components into my demo environment.  I'd love to show this as part of the Power Platform story.  I came from a competitor and governance and CI/CD DevOps processes and pipelines were very central to the low code storyline.

Basically, my issues is that the core load fails...not really any good reason why from what I can see.  The log doesn't seem to share anything super helpful.  I've tried loading this on both an existing and brand new environment...both fail.

I've attached a video of the last portion of the load where it fails plus the log.

[Center of Excellence - Core Components_import.txt](https://github.com/microsoft/coe-starter-kit/files/8232817/Center.of.Excellence.-.Core.Components_import.txt)

https://user-images.githubusercontent.com/90707284/157888926-4b2333b7-c927-4e6b-9af8-91e362c44e06.mp4

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

March 2022

### What app or flow are you having the issue with?

Not quite sure.  Issue is happening on the Core Components load

**Comment by manuelap-msft** [comment]
Hello, can you go to Solution History in this tab and click on the failure there, it sometimes gives a bit more information on why the failure occured. Thank you

**Comment by billwhalenmsft** [comment]
Thank you.  It looks like one of my keys is invalid.  On the history tab....it says the following - "Import failed: Invalid format of KeyVault secret reference found "KEYVALUE"

I did verify the keys that I'm using are setup via the instructions.  I was inputting the key generated from the app registration into this field - Command Center - Client Azure Secret (described as "AZURE VERSION".  I didn't input anything into the "Command Center - Client Secret" field that's described as a "TEXT VERSION".

Is that correct?

**Comment by manuelap-msft** [comment]
For the Command Center - Client Azure Secret, I would recommend leaving that empty on import and then modifying it afterwards to connect it with an Azure KeyVault. To edit afterwards

1. Navigate to make.powerapps.com > Solutions
2. Select the Default solution
3. Filter to view only environment variables
4. Update all secret values: 
   ![image](https://user-images.githubusercontent.com/55705669/157901010-da5b3a27-def4-448c-be89-1617015dbd8e.png)

This should work. We have had a few people have issues by providing an ID during import, but editing afterwards worked for them.

Thank you
Manuela

**Comment by Jenefer-Monroe** [comment]
nope, you should be good to go with just adding that env var afterwards.

**Comment by billwhalenmsft** [comment]
The import is running much longer...so good sign so far.

**Comment by billwhalenmsft** [comment]
Worked!  Thanks Jenefer-Monroe

**Comment by manuelap-msft** [comment]
Wonderful, thanks for letting us know! We've also updated the Description field of this enviornment variable to highlight the above information.

**Comment by zshane15** [comment]
I believe if you put them in this format when importing it should work for you:

`/subscriptions/[ subscriptionId ]/resourceGroups/[ resourceGroupName ]/providers/Microsoft.KeyVault/vaults/[ vaultName ]/secrets/[ secretName ]`

I experienced the same issue and just left the variables blank when importing. After a successful import, I refreshed and the banner said there was an environment variable needing an update.

![image](https://user-images.githubusercontent.com/36671125/206752459-96c0e20b-4332-429f-a8de-118df21e4a0c.png)

When I tried to paste in just the secret name, it gave me an error and referenced the format above. After putting them in that format, it worked. I believe the Default Solution does the exact same thing, it just breaks out the parts of the URL in the panel when entering the values.
